internal/pkg/git/git_test.go

```diff
@@ -0,0 +1,20 @@
+package git
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+)
+
+func TestGit_CheckoutTemplateRepository(t *testing.T) {
+	t.Parallel()
+
+	// should be always true as git is available in the container running the tests
+	assert.True(t, Available())
+
+	fs, err := CheckoutTemplateRepository("https://github.com/keboola/keboola-as-code-templates.git", "main", log.NewDebugLogger())
+	assert.NoError(t, err)
+	assert.True(t, fs.Exists("manifest.json"))
```

Comment:
Tu je chyba, v repository musi existovat `.keboola/repository.json`:
https://github.com/keboola/keboola-as-code/tree/main/test/template-use/in/repository

Reply:
A jejda, i v tom repozitáři (https://github.com/keboola/keboola-as-code-templates)? Tak to musím fixnout.

Reply:
Jj, to si tam dal tusim ty.

Reply:
Ide len o tento test, ... zatial sme ten default repositar nepouzivali.
Ak ho chces takto testovat, tak musis fixnut to repo.

Reply:
Jo jo, teď jsem se snažil marně dohledat proč, jestli jsem to někde četl nebo co. Asi jsem jen nevěděl jak to bude vypadat, tak jsem to tam dal takhle tak jak mě napadlo. 😃 Fixnu.